### PR TITLE
feat: include GAMMA in gateway api page

### PR DESCRIPTION
**Pull request by paularah**
#522

### CODE REVIEW

File: src/pages/use-cases/gateway-api.jsx [hunk 1]
```diff
@@ -45,13 +45,21 @@ const sectionContent3 = {
     'Cilium aims to simplify operations by reducing the number of cloud native networking, observability, and security tools required. It natively supports not just the Gateway API but also features like a sidecar-less Service Mesh, Transparent Encryption, Network Policies, and built-in observability with Hubble. This approach makes life easier for cluster operators, who only need a single tool to manage and upgrade for many, if not all, of their Kubernetes networking needs​​.',
   videoSrc: 'https://www.youtube.com/embed/48ej2qIP92Y',
 };
+const sectionContent4 = {
+  title: 'Gateway API for Service Mesh (GAMMA)',
+  imageRight: false,
+  description:
+    'By leveraging the standard Kubernetes Gateway API for Mesh Management and Administration(GAMMA), Cilium can be used for sophisticated East-West traffic management within the cluster. This is a step forward from its pre-existing mechanism, which relies on customizing the onboard envoy proxy. Cilium support for GAMMA translates into simplicity for operators while providing advanced traffic management.',
```
Comment (xmulligan) [inline src/pages/use-cases/gateway-api.jsx:52]:
```suggestion
    'By leveraging the standard Kubernetes Gateway API for Mesh Management and Administration(GAMMA), Cilium can be used for sophisticated East-West traffic management within the cluster. Cilium support for GAMMA translates into simplicity for operators while providing advanced traffic management.',
```

Comment (xmulligan) [reply src/pages/use-cases/gateway-api.jsx:52]: can you also add one more sentence here?

File: src/pages/use-cases/gateway-api.jsx [hunk 2]
```diff
@@ -45,13 +45,21 @@ const sectionContent3 = {
     'Cilium aims to simplify operations by reducing the number of cloud native networking, observability, and security tools required. It natively supports not just the Gateway API but also features like a sidecar-less Service Mesh, Transparent Encryption, Network Policies, and built-in observability with Hubble. This approach makes life easier for cluster operators, who only need a single tool to manage and upgrade for many, if not all, of their Kubernetes networking needs​​.',
   videoSrc: 'https://www.youtube.com/embed/48ej2qIP92Y',
 };
+const sectionContent4 = {
+  title: 'Gateway API for Service Mesh (GAMMA)',
+  imageRight: false,
+  description:
+    'By leveraging the standard Kubernetes Gateway API for Mesh Management and Administration(GAMMA), Cilium can be used for sophisticated East-West traffic management within the cluster. Cilium support for GAMMA translates into simplicity for operators while providing advanced traffic management.',
+  videoSrc: 'https://www.youtube.com/embed/OAftVdKllak',
```
Comment (xmulligan) [inline src/pages/use-cases/gateway-api.jsx:53]:
```suggestion
  videoSrc: 'https://youtu.be/DYenSJoEVwY?si=J-qKWqJcVM-IAnQG',
```

Comment (paularah) [reply src/pages/use-cases/gateway-api.jsx:53]: I thought the general rule of thumb here is a preference for content from the Cilium community?

Comment (xmulligan) [reply src/pages/use-cases/gateway-api.jsx:53]: good point, I was thinking about using the shorter video, but I can see your point

**Pull request merged**